coroutines-extra/src/test/scala/org/coroutines/extra/async-await-tests.scala

```diff
@@ -1,56 +1,29 @@
-package org.coroutines
+package org.coroutines.extra
```

Comment:
Cool, thanks a lot!

On July 10, 2016 7:49:17 PM GMT+02:00, Jess Smith <notifications@github.com> wrote:
>> @@ -1,56 +1,29 @@
>> -package org.coroutines
>> +package org.coroutines.extra
>
>Yep- just didn't get to them yesterday. On it right now.
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/storm-enroute/coroutines/pull/23/files/124c3a1d8e475c60c2dcb1489f974d85de4badfb#r70185792

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.